<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item row="3" column="2">
           <widget class="QPushButton" name="FileAPIDecrypt">
            <property name="text">
             <string>File API Decrypt</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEdit_redirectUrl">
            <property name="text">
             <string>com.microsoft.rms-sharing-for-android://authorize</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_clitntId">
            <property name="text">
             <string>Client ID</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="FileAPIEncrypt">
            <property name="text">
             <string>File API Encrypt (templates)</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_clientEmail">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="7" column="2">
           <widget class="QPushButton" name="certificatesPathButton">
            <property name="text">
             <string>Browse...</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" rowspan="4">
           <widget class="QLabel" name="label_redirectUrl">
            <property name="text">
             <string>Redirect URI</string>
            </property>
           </widget>
          </item>
          <item row="6" column="2">
           <widget class="QPushButton" name="fromPFILEButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Decrypt PFILE</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLineEdit" name="ineEdit_certificatesPath">
            <property name="text">
             <string>./certificates/</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="encryptPFILERightsButton">
            <property name="text">
             <string>Encrypt  Sync (rights)</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_resourseId">
            <property name="text">
             <string>Resource ID</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1" rowspan="2">
           <widget class="QLineEdit" name="lineEdit_resourseId">
            <property name="text">
             <string>https://api.aadrm.com</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_certificatesPath">
            <property name="text">
             <string>Certificates path</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_clientId">
            <property name="text">
             <string>ECAD3080-3AE9-4782-B763-2DF1B1373B3A</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="encryptPFILETemplatesButton">
            <property name="text">
             <string>Encrypt Async (templates)</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_clientEmail">
            <property name="text">
             <string>Client Email</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QPushButton" name="FileAPIEncryptRights">
            <property name="text">
             <string>File API Encrypt (rights)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QTextBrowser" name="textBrowser">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit_clientId</tabstop>
  <tabstop>lineEdit_clientEmail</tabstop>
  <tabstop>lineEdit_redirectUrl</tabstop>
  <tabstop>lineEdit_resourseId</tabstop>
  <tabstop>ineEdit_certificatesPath</tabstop>
  <tabstop>textBrowser</tabstop>
  <tabstop>encryptPFILETemplatesButton</tabstop>
  <tabstop>encryptPFILERightsButton</tabstop>
  <tabstop>fromPFILEButton</tabstop>
  <tabstop>certificatesPathButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
